pithos/PreferencesPithosDialog.py

```diff
@@ -111,12 +111,19 @@ class PreferencesPithosDialog(Gtk.Dialog):
     control_proxy_entry = GtkTemplate.Child()
     control_proxy_pac_entry = GtkTemplate.Child()
     explicit_content_filter_checkbutton = GtkTemplate.Child()
+    use_downloadbuffering_checkbutton = GtkTemplate.Child()
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, use_header_bar=1, **kwargs)
         self.init_template()
 
         self.settings = Gio.Settings.new('io.github.Pithos')
+        self.settings.bind(
+            'use-downloadbuffering',
+             self.use_downloadbuffering_checkbutton,
+             'active',
```

Comment:
__[Flake8]__

continuation line unaligned for hanging indent


<!-- Comment by [SideCI](https://sideci.com) -->